starlette_admin/contrib/sqla/converters.py

```diff
@@ -123,6 +124,12 @@ class ModelConverter(BaseSQLAModelConverter):
     def _field_common(
         cls, *, name: str, column: Column, **kwargs: Any
     ) -> Dict[str, Any]:
+        if type(column) is Label:
+            return {
+                "name": name,
+                "help_text": name,
```

Comment:
The `help_text` is only required for columns with a `comment` attribute, so we can add a condition to check for it.